packages/database/src/core/PersistentConnection.ts

```diff
@@ -236,9 +236,9 @@ export class PersistentConnection extends ServerActions {
 
     req[/*hash*/ 'h'] = listenSpec.hashFn();
 
-    this.sendRequest(action, req, (message: { [k: string]: any }) => {
-      const payload: any = message[/*data*/ 'd'];
-      const status: string = message[/*status*/ 's'];
+    this.sendRequest(action, req, (message: { [k: string]: unknown }) => {
+      const payload: unknown = message[/*data*/ 'd'];
+      const status: string = message[/*status*/ 's'] as string;
```

Comment:
Nit: You can probably drop the explicit type in the declaration.

Reply:
Done.